pkg/bpf/decoder.go

```diff
@@ -66,7 +66,7 @@ func decode(inst linux.BPFInstruction, line int, w *bytes.Buffer) error {
 	case Misc:
 		err = decodeMisc(inst, w)
 	default:
-		return fmt.Errorf("invalid BPF instruction: %v", inst)
+		return fmt.Errorf("invalid BPF instruction: %w", inst)
```

Comment:
You can only use %w for error arguments.

Reply:
I fixed these issues.